src/ZfcRbac/Guard/AbstractGuard.php

```diff
@@ -46,9 +46,9 @@
     /**
      * {@inheritDoc}
      */
-    public function attach(EventManagerInterface $events)
+    public function attach(EventManagerInterface $events, $priority = self::EVENT_PRIORITY)
```

Comment:
because it can be extended/changed if you leave it as is and also it was static before.